test/testcafe/spec/DeviceSSOExtensionView_spec.js

```diff
@@ -89,7 +89,7 @@ test
   .requestHooks(logger, redirectSSOExtensionMock)('with redirect SSO Extension approach, opens the verify URL', async t => {
     const ssoExtensionPage = new BasePageObject(t);
     await ssoExtensionPage.navigateToPage();
-    await ssoExtensionPage.formExists();
+    await t.expect(ssoExtensionPage.formExists()).notOk();
```

Comment:
why `.notOk()` here?

Reply:
This test uses [mock](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/identify-with-apple-redirect-sso-extension.json) that triggers immediate redirect to SSO extension page, so widget should not be rendered